code/modules/awaymissions/corpse.dm

```diff
@@ -21,6 +21,10 @@
 	density = 1
 	anchored = 1
 
+/obj/effect/mob_spawn/New()
+	if(!mob_gender)
+		mob_gender = pick(MALE, FEMALE)
+
```

Comment:
How about linking it to ghost's character setup gender instead of making it random? We don't want to become Rust.